content/chapters/compute/lab/content/processes.md

```diff
@@ -114,17 +114,17 @@ From the outside, it's as if you were running these commands from the terminal.
 Head over to `support/sleepy/sleepy_creator.py`.
 Use `subprocess.Popen()` to spawn 10 `sleep 1000` processes.
 
-1. Now use the same `pstree -pac` command and look for `sleepy_creator.py`.
+1. Now use `ps -e -H -o pid,ppid,cmd | (head -1; grep "python3 sleepy_creator.py")` command and look for `sleepy_creator.py`.
 It is a `python3` process, as this is the interpreter that runs the script, but we call it the `sleepy_creator.py` process for simplicity.
-If you found it, you did something wrong.
-It should be missing.
-Now use `pstree -pac` and look for the `sleep` processes you have just created.
+You shouldn't be able to find it, as the parent process (`sleepy_creator.py`) dies before its children (the 10 `sleep 1000` subprocesses) finish their execution, therefore their parent becoming either `systemd`/`init` or another system process that adopts orphan processes.
+Now use `ps -e -H -o pid,ppid,cmd | (head -1; grep sleep)` and look for the `sleep` processes you have just created.
 
 [Quiz](../quiz/parent-of-sleep-processes.md)
 
-1. Change the code in `sleepy_creator.py` so that the `sleep 1000` processes are the children of `sleepy_creator.py`.
+2. Change the code in `sleepy_creator.py` so that the `sleep 1000` processes are the children of `sleepy_creator.py`.
```

Comment:
Do not use `2.` Use `1. ` but prepend all lines for a list item with three spaces, to make it indent correctly.